### PR TITLE
feat: add `invariant`

### PR DESCRIPTION
**Pull request by Jayllyz**
#136

### CODE REVIEW

File: docs/modules/invariant.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```ts
+import invariant from 'tiny-invariant';
+
+invariant(truthyValue, 'This should not throw!');
+
+invariant(falsyValue, 'This will throw!');
+```
+
+[Project Page](https://github.com/alexreardon/tiny-invariant)
+[NPM](https://www.npmjs.com/package/tiny-invariant)
```
Comment (SuperchupuDev) [inline docs/modules/invariant.md:20]:
```suggestion
[npm](https://www.npmjs.com/package/tiny-invariant)
```

**Review by 43081j — APPROVED** [review]
looks good to me 👍

**Comment by 43081j** [comment]
i think you might need to run `npm run sort-manifests`

File: scripts/sort-manifests.js [hunk 2]
```diff
@@ -20,5 +20,5 @@ for (const manifestName of manifests) {
     return a.moduleName > b.moduleName ? 1 : -1;
   });
 
-  await writeFile(manifestPath, JSON.stringify(manifest, null, 2), 'utf8');
+  await writeFile(manifestPath, JSON.stringify(manifest, null, 2) + '\n', 'utf8');
```
Comment (43081j) [inline scripts/sort-manifests.js:23]: not sure why this is showing up in the diff 🤔 
any idea?

Comment (Jayllyz) [reply scripts/sort-manifests.js:23]: That's weird, idk

**Review by 43081j — APPROVED** [review]

**Pull request merged**